src/js/apps/globals/nav_app.js

```diff
@@ -310,9 +287,36 @@ export default RouterApp.extend({
     }
   },
   showMainNavDroplist() {
+    const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    const workspaces = currentUser.getWorkspaces();
+
+    const workspacesMenu = new Backbone.Collection(
+      workspaces.map(workspace => {
+        return {
+          id: workspace.id,
+          onSelect() {
+            Radio.trigger('event-router', `workspace:${ workspace.get('slug') }`);
+          },
+          text: workspace.get('name'),
+          icon: {
+            type: 'far',
+            icon: 'window',
+          },
+        };
+      }),
+    );
+
+    const workspacesMenuList = {
+      collection: workspacesMenu,
+      headingText: i18n.mainNavDroplist.workspacesHeading,
+      headingClassName: 'app-nav__picklist-heading',
+    };
+
     this.showChildView('navMain', new MainNavDroplist({
-      collection: appNavMenu,
+      lists: [workspacesMenuList],
```

Comment:
you can just pass this as `collection: workspacesMenuList`
The droplist only needs a list if there's more than one.